C7/UIElements/Theme.cs

```diff
@@ -0,0 +1,15 @@
+using Godot;
+
+public static class ThemeFactory {
+
+	static ThemeFactory() {
```

Comment:
The idea is to start putting all of the hardcoded theme defaults here. In the future, ThemeFactory can read from a config file so it is customizable (ie. for modding).

Reply:
love it